fix: Add option to disable stacktrace feature (#3345)
aws/s2n-tls: CMakeLists.txt

version of libcrypto by interning the code and hiding symbols. This also enables s2n-tls to be
loaded in an application with an otherwise conflicting libcrypto version." OFF)
option(S2N_LTO, "Enables link time optimizations when building s2n-tls." OFF)

> option(S2N_NO_STACKTRACE "Disables stacktrace functionality in s2n-tls." OFF)

Reviewer: having "no" in feature names makes things more difficult to follow. I would rather have the default be `ON`

```suggestion
option(S2N_STACKTRACE "Enables stacktrace functionality in s2n-tls." ON)
```

Author: That makes sense! Fixed in f620d574032bd9f1e4b073670e2c273870f1baa8
